config/kafka/config.go

```diff
@@ -25,6 +25,14 @@ func Configure(p *config.Provider) {
 		r.References["broker_node_group_info.security_groups"] = config.Reference{
 			Type: "github.com/upbound/provider-aws/apis/ec2/v1beta1.SecurityGroup",
 		}
+		r.References["configuration_info.arn"] = config.Reference{
+			Type:      "Configuration",
+			Extractor: common.PathARNExtractor,
+		}
+		r.References["configuration_info.revision"] = config.Reference{
+			Type:      "Configuration",
+			Extractor: "GetConfigurationRevision()",
```

Comment:
```suggestion
			Extractor: `github.com/upbound/upjet/pkg/resource.ExtractParamPath("latest_revision",true)`,
```
We have a [function](https://github.com/upbound/upjet/blob/0154ca2d806f389c294ed475d2535a96ddb4c3c0/pkg/resource/extractor.go#L34) for such situations, could you please try the configuration above?

Reply:
Yes! I thought I remembered seeing something like that, and looked for it, but couldn't find it. This is much better.